django/cantusdb_project/main_app/templates/ci_search.html

```diff
@@ -42,12 +42,12 @@
             var cantus_id = document.getElementById(rowId).cells[1].innerText;
             var genre = document.getElementById(rowId).cells[2].innerText;
             var full_text = document.getElementById(rowId).cells[3].innerText;
-
-            // get the option value corresponding to the genre name
-            const genreId = Array.prototype.find.call(opener.document.getElementById('id_genre').children, ({ textContent }) => textContent === genre);
+            var genres = {{ genres|safe }}; // genres contains "id" and "name" of all Genre objects in the database
+            var genreObj = genres.find(item => item.name === genre);
+            var genreId = genreId = genreObj ? genreObj.id : null;
```

Comment:
how many of these could be `const`s?

Reply:
After testing this, I believe that they must remain as `var`s because if the user uses the chant input tool once it works fine but if they try and use it again and change the chant the variables will be locked to the previous chant's stored variables.